Add content about HumbleData workshop (#293)
EuroPython/website: data/pages-content/humble-data.md

---

> title: Beginners' Day - Humble Data Workshop

Reviewer: ```suggestion
Humble Data Workshop
```

subtitle:
  Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking first steps are very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!

> ---

Reviewer: If you want to add a custom social card, feel free to add the file to https://github.com/EuroPython/website/tree/main/social-cards. 
You then should add 
`social_card: card-filename.png` (see this as an example: https://raw.githubusercontent.com/EuroPython/website/main/data/pages-content/pyladies-workshop.md)

assets: https://drive.google.com/drive/folders/1cmOLAsNUSO2L8FCYmH5TZS5gon79buPY?usp=sharing

Author: I created the social card and added it to the post. I don't have permission to add the file to the GDrive folder. 😭

> ## Beginners' Day - Humble Data Workshop ##

Reviewer: ```suggestion
## Humble Data Workshop ##
```

*WE ARE BACK!*
Following the success of several Beginners Data Workshop at PyCon US, Ghana, Namibia, Africa and PyData Global, we are glad to have it again at EuroPython!

👉🏽 [APPLY HERE TO ATTEND THIS WORKSHOP](https://forms.gle/jrF8uSb8w5tMhAyP8) (application deadline is 3rd July, 2023) 👈

## Learn Python for Data Science in this Beginners' Day Workshop ##
Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking first steps are very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!

“We want our community to be more diverse: whoever you are, and whatever your background, we welcome you.” – Python Software Foundation’s Diversity Statement

> We invite those from under-represented groups to apply to join us on Monday 17th July 2023 for the EuroPython Beginners' Day - Humble Data Workshop. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook that help with data analysis.

Reviewer: ```suggestion
We invite those from under-represented groups to apply to join us on Monday 17th July 2023 for the EuroPython - Humble Data Workshop. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook that help with data analysis.
```

You will be divided into small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. You can expect plenty of exercises, quizzes and games, and inspiring talks from those who were once in your shoes.

## WHEN? ##
Monday, 17th July 2023 (time and room to be determined).

## WHERE? ##
EuroPython (see [here](/where)).

## WHO CAN JOIN? ##

> Anyone with a EuroPython [Conference Ticket or Combined Ticket](https://ep2022.europython.eu/tickets#ticket-types).

Reviewer: ```suggestion
Anyone with a EuroPython  in-person [Conference , Tutorial or Combined Ticket](https://ep2023.europython.eu/tickets#ticket-types).
```

Reviewer: @laisbsc this still needs to be changed: 2022-2023

Author: Awesome catch! TY.

## WHAT DO I NEED TO BRING? ##
You will need a computer or tablet with internet access. Nothing else!
 Our mentors will help you set up on the day.

## WHAT DO I NEED TO KNOW? ##
If you have zero programming experience, this workshop is made for you! If you already have some experience but are new to Python, we’ve got you covered too – the materials are designed for attendees to progress through at their own pace.

## HOW IS THIS POSSIBLE? ##
This workshop is being run by data scientists and software engineers dedicated to the spirit of open source. Every day we benefit from the hard work of others when we use free open tools. This is our way of giving back. We thank EuroPython for their support in facilitating this workshop. ❤️

If you would like to attend this workshop, [please apply here](https://forms.gle/jrF8uSb8w5tMhAyP8). Deadline: 3rd July, 2023.

## Mentors ##

> We are looking for volunteers to help mentor the attendees. There is no need to be an expert in programming, data science or Python – we are teaching the basics. But you do need: <br>

Reviewer: ```suggestion
We are looking for volunteers to help mentor the attendees. There is no need to be an expert in programming, data science or Python – we are teaching the basics. But you do need:

```

Author: Removed.

👉🏽 Lots of patience, smiles and a *friendly attitude*.

👉🏽 The ability to answer questions in a beginner-friendly way, even if it means the explanation is not technically precise.

👉🏽 Curiosity.

👉🏽 Respect.

👉🏽 Will to learn and to teach the mentees.

> In return, you will meet new interesting people outside your usual *bubble*, gain eternal gratitude from a group of attendees, and help to break down stereotypes around what a developer “looks like”. <br>

Reviewer: ```suggestion
In return, you will meet new interesting people outside your usual *bubble*, gain eternal gratitude from a group of attendees, and help to break down stereotypes around what a developer “looks like”.

```

## Learn Python for Data Science in the Workshop ##
Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking first steps are very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!

“We want our community to be more diverse: whoever you are, and whatever your background, we welcome you.” – Python Software Foundation’s Diversity Statement

We invite those from under-represented groups to apply to join us on Monday 17th July 2023 for the Humble Data Workshop at EuroPython. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook that help with data analysis.

You will be divided into small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. You can expect plenty of exercises, quizzes and games, and inspiring talks from those who were once in your shoes.

## WHEN? ##
Monday, 17th July 2023 (time and room to be determined).

## WHERE? ##

> EuroPython (see [here](/where)).

Reviewer: Better to point to /where#prague-congress-centre---conference-venue
Since the where page has Prague, PCC and VŠE info.

Author: Done :)
